scaleway/resource_object_bucket.go

```diff
@@ -275,6 +281,13 @@ func resourceScalewayObjectBucketDelete(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(err)
 	}
 
+	if d.Get("force_destroy").(bool) {
+		err = deleteS3Objects(ctx, s3Client, bucketName)
```

Comment:
Hello @Codelax, please consider the versions if there are

Reply:
You can check a similar implementation [here](https://github.com/hashicorp/terraform-provider-aws/blob/26f2311f79416f5d37c439288a7287a3b6a56c26/internal/service/s3/bucket.go#L1305)